packages/editor/src/components/blocks/datatablecolumn/DataTableColumn.tsx

```diff
@@ -18,21 +22,31 @@ export const DataTableColumnComponent: ComponentConfig<DataTableColumnProps> = {
   description: 'A Column for the DataTable',
   defaultProps: defaultDataTableColumnProps,
   render: props => <UiBlock {...props} />,
-  create: ({ label, value }) => ({ ...defaultDataTableColumnProps, header: label, value }),
+  create: ({ label, value }) => ({ ...defaultDataTableColumnProps, header: label, value, filterable: false, sortable: false }),
```

Comment:
have a look at the other creates, the filterable and sortable are already here, as they are in the defaultDataTableColumnProps